src/app/authentication/pages/login/login.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="content-wrapper">
+  <div class="row">
+    <div class="col col-md-3 mx-auto text-center">
+      <img src="assets/images/logo.svg" class="header__logo"/>
+      <p class="header__subtitle">Sign in to Nimble</p>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col col-md-3 mx-auto">
```

Comment:
@carryall Should we take care more about grid system in some screens such as this?
![Screen Shot 2563-11-24 at 15 13 49](https://user-images.githubusercontent.com/749672/100066422-a9c98e80-2e67-11eb-9779-d83cdbf16947.png)